src/neuvector/tasks.yaml

```diff
@@ -41,6 +41,11 @@ tasks:
             protocol: https
             address: neuvector.admin.uds.dev
             code: 200
+      - description: Wait for NeuVector Job Completion
+        cmd: |
+          if kubectl get jobs -n neuvector --no-headers 2>/dev/null | grep -q .; then
```

Comment:
./zarf tools kubectl?

Reply:
ty yep.

Reply:
Updated to `./uds zarf tools kubectl` based on usage in other tasks currently. Not a huge issue either way since this is just validations and not in the `zarf.yaml` but this is more portable 🚀